languages/javalang/builder-renderer/generated-code/src/test/java/com/commercetools/ModelSerializationTest.java

```diff
@@ -0,0 +1,164 @@
+package com.commercetools;
+
+import com.commercetools.models.Category.*;
+import com.commercetools.models.Common.*;
+import com.commercetools.models.Type.*;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+
+public class ModelSerializationTest {
+    
+    @Test
+    public void serializeCategoryDraftToJson() {
+        LocalizedString localizedString = new LocalizedStringImpl();
+        localizedString.setValue("test-key", "test-value");
+        String key = "test-key";
+        String id = "test-id";
+        String randomString = "test-string";
+        
+        CategoryResourceIdentifier resourceIdentifier = new CategoryResourceIdentifierImpl();
+        resourceIdentifier.setId(id);
+        resourceIdentifier.setKey(key);
+
+        FieldContainer fieldContainer = new FieldContainerImpl();
```

Comment:
We didn't you use a builder here?

Reply:
FieldContainer is not a draft class so it doesn't have a builder.